src/meta/src/hummock/compaction/compaction_config.rs

```diff
@@ -35,11 +35,10 @@ impl CompactionConfigBuilder {
         Self {
             config: CompactionConfig {
                 max_bytes_for_level_base: DEFAULT_MAX_BYTES_FOR_LEVEL_BASE,
-                max_bytes_for_level_multiplier: 10,
+                max_bytes_for_level_multiplier: 5,
```

Comment:
why did we change the default value of `max_bytes_for_level_multiplier`?

is any benchmark result for this?

Reply:
https://www.notion.so/risingwave-labs/Exp-records-96630a04277d4c4bb7a8e7d5227bf5bf